server/src/main/java/org/opensearch/action/admin/cluster/node/stats/NodeStats.java

```diff
@@ -308,6 +320,11 @@ public SearchBackpressureStats getSearchBackpressureStats() {
         return searchBackpressureStats;
     }
 
+    @Nullable
+    public ClusterManagerThrottlingStats getClusterManagerThrottlingStats() {
```

Comment:
tests for throttling stats?

Reply:
I checked the UTs for other stats and based on that added relevant UTs for this stat too.